15 LoginForm/src/pages/common/centered.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react"
```

Comment:
Missing in readme.md the containerComponent refactor (could be added as last step)